recipes/glib/all/conanfile.py

```diff
@@ -198,7 +198,10 @@ def package_info(self):
                 self.cpp_info.components["gio-2.0"].requires.append("libmount::libmount")
             if self.options.with_selinux:
                 self.cpp_info.components["gio-2.0"].requires.append("libselinux::libselinux")
-        if self.settings.os != "Windows":
+        if self.settings.os == "Windows":
+            self.cpp_info.components["gio-windows-2.0"].requires = ["gobject-2.0", "gmodule-no-export-2.0", "gio-2.0"]
+            self.cpp_info.components["gio-windows-2.0"].includedirs = [os.path.join('include', 'gio-win32-2.0')]
+        else:
             self.cpp_info.components["gio-unix-2.0"].libs = ["gio-2.0"]
```

Comment:
Sorry, I've approved, but just one question: there is a lib file for `gio-unix-2.0` but not for `gio-windows-2.0`?

Reply:
that's a wrong copy/paste in the last PR, I'm removing it